plugins/by-name/wezterm/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "wezterm";
+  originalName = "wezterm.nvim";
+  package = "wezterm-nvim";
+
+  maintainers = [ lib.maintainers.samos667 ];
+
+  extraOptions = {
+    weztermPackage = lib.mkPackageOption pkgs "wezterm" {
+      nullable = true;
+      default = null;
+      example = [ "wezterm" ];
+    };
```

What's the best default here? The package or null?

I'm assuming most users who enable this plugin will already have wezterm installed 🤔

---

Usually we set the package.

---

This would achieve that:

```suggestion
    weztermPackage = lib.mkPackageOption pkgs "wezterm" {
      nullable = true;
    };
```

Is there any reason not to do that here?